tests/testthat/test-epinowcast.R

```diff
@@ -11,6 +11,13 @@ if (not_on_cran() & on_ci()) {
   }
 }
 
+# A function to filter data used in test suite
+run_window_filter <- function(x){
```

Comment:
```suggestion
run_window_filter <- function(x) {
```
pretty sure the linter will complain about this one. did you `lintr::lint()` to check?

Reply:
probably also make this slightly more flexible, so you can use it later w/ diff limits?

Reply:
Good call. Updated the function arguments to allow passing different limits as suggested (see [48c3a38](https://github.com/epinowcast/epinowcast/pull/402/commits/48c3a3850c2e715200f3b4633a2d161bc8baebcb)